src/stylesheets/elements/_headings.scss

```diff
@@ -193,6 +212,13 @@ footer h2 {
   }
 }
 
+.track--alternative__light {
```

Comment:
By changing `.track__light` to `.track--light`, we won't need this anymore. We can instead use them together in our HTML like this,

```
<ul class="track--alternative track--light">
  ...
</ul>
```